README.md

```diff
@@ -2,16 +2,39 @@
 
 [![CircleCI](https://circleci.com/gh/invainn/ARIA/tree/master.svg?style=svg)](https://circleci.com/gh/invainn/ARIA/tree/master)
 
-## Description
+## Project Description
 ---
-TODO
+ARIA (Administration, Registration, and Information Assistant) is a web application solution for the Northern Nevada Music Teachers Association (NNMTA). Previous versions of ARIA were hosted on WordPress and contained performance issues as well as functional problems with certain features for both customers and adminstrators.
+
+This version of ARIA (version 3.0) attempts to solve problems with previous iterations of ARIA by creating a full-stack web application built in React.js, Node.js, and PostgreSQL, along with many other tools. Building the web application from the ground up allows our team to have more control over development and addressing the needs of the customer.
+
+
+This project is being developed to satisfy the CS 425 - Software Engineering course requirements at the University of Nevada Reno. Our sponsoring faculty member from the Computer Science and Engineering Department is Dr. Frederick Harris, and our Ph.D. advisor/mentor is Jalal Kiswani.
 
 ## Team Members
 ---
 * Anthony Bennett
 * Nikkolas Irwin
 * Kevin Carlos
 
+## Running the application
+---
+There are many tools utilized in this project.
+
+To run ARIA you will need to do the following:
+
+1. Install both the Docker Engine and Docker Compose from Docker's website: https://www.docker.com/
+
+2. From the project directory, start up the application by running the two following commands:
+
+	a) docker-compose build <br />
+	b) docker-compose up <br />
+
+These commands will build the image and start the container.
+
 ## Libraries/Tools
 ---
-TODO
+Docker <br />
```

Comment:
Need to add 
---
* Redux
* Redux Thunk
* Material-UI 
* Express
* Axios
* Jest
* ESLint
* PostgreSQL